docs/mine/lotus/miner-wallets.md

```diff
@@ -1,145 +1,149 @@
+During miner initialization, a _miner actor_ is created on the chain, and this actor gives the miner its ID `t0...`. The miner actor is in charge of collecting all the payments sent to the miner. For example, when a payment is sent for honoring the different types of deals, that payment goes to the miner actor, not the miner itself.
 
-The Lotus Miner daemon, in turn, performs the operations as required by the network during and can use different Lotus Node wallets to pay the fees or interact with the _miner actor_. Information on how to manage Lotus wallets [can be found here](../../get-started/lotus/send-and-receive-fil.md).
+The Lotus Miner daemon performs the operations required by the network and can use different Lotus node wallets to pay the fees or interact with the _miner actor_. Check out the [Lotus Getting Started guide](../../get-started/lotus/send-and-receive-fil.md) for more information on how to manage Lotus wallets.
 
 ::: tip
-We use the terms _wallet_ and _address_ rather interchangeably in this document. A Lotus _wallet_ is identified by its _address_.
+Each node can only have one wallet, but each wallet can have multiple addresses.
 :::
 
-The currently configured wallets used by a miner can be listed with:
+The currently configured addresses used by a miner can be listed with:
 
 ```sh
 lotus-miner actor control list
 ```
 
-The different types of wallets associated to a miner are described below:
-
-[[TOC]]
+The different types of addresses associated with a miner are described below:
 
 ## The owner address
 
-The owner address corresponds to a Lotus Node wallet address that is provided during the [miner initialization](miner-setup.md). The _owner address_ is only needed when:
+The owner address corresponds to a Lotus node address provided during the [miner initialization](miner-setup.md). The _owner address_ is only needed when:
 
-- Changing the owner or the worker address in the _miner actor_
-- Withdrawing balance from the _miner actor_
-- Submit _WindowPoSts_ (**unless _control addresses_ are defined and have enough balance**, see below)
+- Changing the owner or the worker address in the _miner actor_.
+- Withdrawing balance from the _miner actor_.
+- Submit _WindowPoSts_, **unless _control addresses_ are defined and have enough balance** (continued below).
 
-The wallet associated to the _owner address_ is designed to be kept offline as a cold wallet, given that it should not be used often and it is critical to safeguard the miner's funds. In production environments, we strongly recommend using separate _owner_ and _worker_ addresses.
+The wallet associated with the _owner address_ is designed to be kept offline as a cold wallet since it should not be used often. It is critical to safeguard the miner's funds. In production environments, we strongly recommend using separate _owner_ and _worker_ addresses.
```

Comment:
@hsanjuan, I'm confused about this section.

> The wallet associated with the _owner address_ is designed to be kept offline as a cold wallet.

This makes sense. But if we replace _The wallet associated_ with _The address associated_ then it doesn't, since you can't have a _cold-address_. Can you?

Reply:
I think we want to say something like `The address chosen to be the miner's _owner address_ is designed to be mostly kept offline or otherwise backed by [hardware wallet](link to ledger guide)`.

The address will need to be available to lotus in some moments when performing some of the actions but otherwise it can be exported and removed.

Reply:
```suggestion
The address chosen to be the miner's _owner address_ is designed to be kept offline in _cold storage_, or backed up by a [hardware wallet](../../get-started/lotus/ledger). In production environments, we strongly recommend using separate _owner_ and _worker_ addresses.
```